### ISSUE TITLE
discuss using Rob's CMDI browser for Translatin database with Rob

### ISSUE THREAD

**Issue by hennie** [knaw-huc/textannoviz#163]

**Comment by svandaalen** [comment]
Moet dit issue bij TAV blijven staan of hoort deze eigenlijk in https://github.com/knaw-huc/team-text-backlog?

**Comment by hennie** [comment]
Nee, moet team-text-backlog zijn, en lijkt dat ook te zijn. Ik weet niet waar 'textannoviz' vandaan komt. Ik probeer het te veranderen
